ui/ducks/send/send.js

```diff
@@ -2042,6 +2094,42 @@ export function updateGasPrice(gasPrice) {
   };
 }
 
+export function updateSendQuote(
+  isComputingSendGasLimit = true,
+  isRefreshingQuotes = false,
+) {
+  return async (dispatch, getState) => {
+    const state = getState();
+
+    const draftTransaction =
+      state[name].draftTransactions[state[name].currentTransactionUUID];
+
+    const isSwapAndSend =
+      draftTransaction?.sendAsset?.details?.address !==
+      draftTransaction?.receiveAsset?.details?.address;
```

Comment:
Extract to a selector?

Reply:
Selectors are created later in the file (i.e., after usage)

Reply:
Good point. Maybe an internal method then? This condition is used multiple times so that would keep them in sync and reusable